.mergify.yml

```diff
@@ -0,0 +1,13 @@
+pull_request_rules:
+  - name: Automatic strict merge
+    conditions:
+      - base=master
+      - assignee=box-merge
+      - "#approved-reviews-by>=1"
+      - "#changes-requested-reviews-by=0"
+      - status-success=continuous-integration/travis-ci/pr
```

Comment:
We can also add `#review-requested=0` to make sure that the review groups added by CODEOWNERS has been satisfied